src/main/scala/org/broadinstitute/hail/methods/LinearRegression.scala

```diff
@@ -74,13 +74,15 @@ class LinRegBuilder extends Serializable {
     this
   }
 
-  def stats(y: DenseVector[Double], n: Int): Option[(SparseVector[Double], Double, Double)] = {
+  def stats(y: DenseVector[Double], n: Int, minAC: Int): Option[(SparseVector[Double], Double, Double)] = {
+    require(minAC > 0)
+
     val missingRowIndicesArray = missingRowIndices.result()
     val nMissing = missingRowIndicesArray.size
     val nPresent = n - nMissing
 
-    // all HomRef | all Het | all HomVar
-    if (sumX == 0 || (sumX == nPresent && sumXX == nPresent) || sumX == 2 * nPresent)
+    // all HomRef or all Missing or AC < minAC || all HomVar || all Het
+    if (sumX < minAC || sumX == 2 * nPresent || (sumX == nPresent && sumXX == nPresent))
```

Comment:
in cases like this people often use named variables rather than comments (since comments are generally believed to be ignored by folks editing code):
```scala
val allHomRef = sumX < minAC
val allMissing = sumX == 2 * nPresent
// etc.
```